packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterConfirmSignInRequest.kt

```diff
@@ -29,6 +31,15 @@ data class FlutterConfirmSignInRequest(val map: HashMap<String, *>) {
     if(rawOptions?.get("clientMetadata") != null)
       options.metadata(rawOptions["clientMetadata"] as HashMap<String, String>);
 
+    if(rawOptions?.get("userAttributes") != null) {
+      var attributes = mutableListOf<AuthUserAttribute>();
+      var rawAttributes = rawOptions["userAttributes"] as HashMap<String, String>;
+      rawAttributes.forEach { (key, value) ->
+        attributes.add(createAuthUserAttribute(key, value))
+      }
```

Comment:
Nit: Rather than declaring a mutable list and then adding elements in, would it be possible to just map the userAttributes? E.g.
```suggestion
    if(rawOptions?.get("userAttributes") != null) {
      val rawAttributes = rawOptions["userAttributes"] as HashMap<String, String>
      val attributes = rawAttributes.map { (key, value) ->
        createAuthUserAttribute(key, value)
      }
```

Also noticed this is slightly different from the Swift implementation where it seemed ok to have the attributes be an empty list. If it's ok here too, we can make this even more concise

```suggestion
    val rawAttributes = rawOptions?.get("userAttributes") as HashMap<String, String>? ?: emptyMap<String, String>()
    val attributes = rawAttributes.map { (key, value) ->
      createAuthUserAttribute(key, value)
    }
```